browser/ui/color/brave_color_mixer.cc

```diff
@@ -279,10 +280,7 @@ void AddBraveLightThemeColorMixer(ui::ColorProvider* provider,
   mixer[kColorFeaturePromoBubbleCloseButtonInkDrop] = {
       GetToolbarInkDropColor(mixer)};
 
-  mixer[kColorBraveVerticalTabSeparator] = {SkColorSetRGB(0xE2, 0xE3, 0xE7)};
-  mixer[kColorBraveVerticalTabActiveBackground] = {SK_ColorWHITE};
-  mixer[kColorBraveVerticalTabInactiveBackground] = {
-      SkColorSetRGB(0xf3, 0xf3, 0xf3)};
+  tabs::AddBraveVerticalTabLightThemeColorMixer(provider, key);
```

Comment:
Maybe this should be guarded with in `base::FeatureList::IsEnabled(tabs::features::kBraveVerticalTabs)`?

Reply:
Good call!